WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -904,17 +928,19 @@ local function subTypesFor(item, regionType)
       createConditions = function(conditions, item, regionType)
         missingBuffGreyed(conditions, 1, regionType);
       end,
-      data = { inverse = false },
+      data = data,
```

Comment:
Haven't tested this exact code path "Always shows the aura, grey if buff not active." does indeed include a unused glow sub region. So I believe the above code is a left-over?